### ISSUE TITLE
Открытие ссылки в новой вкладке

### ISSUE THREAD

**Issue by Sharzon** [razrabs-media/journal#347]
**Что это решает и поможет?**
Неудобно, что ссылка открывается во вкладке сайта. Приходится либо открывать руками в новой вкладке, либо уходить с сайта. Когда это происходит в середине статьи, это крайне неудобно

**Описание желаемого результата**
При нажатии на любую ссылку из статьи, она открывается в новой вкладке

**Comment by philippranzhin** [comment]
Хм хм хм.

Я обычно делаю ПКМ->открыть в новой вкладке, потому что привык что обычно сайты открывают в этой же — но мне никогда не нужно чтобы открывали в этой же

Никто не знает, откуда такая традиция пошла? Я бы и правда предпочёл по дефолту в новой открывать

**Comment by ninedev-i** [comment]
Средняя кнопка мыши спасает. Вообще я не люблю когда сайт открывает что-то в новой вкладке без моего ведома.

**Comment by philippranzhin** [comment]
Твиттер кстати в новой вкладке открывает. Давайте у себя тоже так сделаем

**Comment by batyshkaLenin** [comment]
target _blank один из инструментов SEO. Когда пользователь уходит с сайта по ссылке, считается что сессия завершена и это ухудшает поведенческие факторы. Вообще, для всех внешних ссылок в статьях нужно использовать rel=”noopener” и rel=”noreferrer”, первое отрубает доступ сайту на который пришли в window.opener, во втором window.referrer.

**Comment by Sharzon** [comment]
Я все могу понять, но какое медиа не возьми, такое поведение, как я описал, является стандартным. Я читаю разрабов в основном с телефона. Там приходится зажимать, ждать. Это раздражает. Даже с компа это неудобно, потому что это нифига неожидаемое поведение. Лично я точно забуду об этой особенности вашего сайта, и это будет вызывать только раздражение. Я все понимаю, что вы особенные, отличные от других, но зачем пользователей напрягать лишний раз, потому что они разработчики?

**Comment by ninedev-i** [comment]
Я понял, речь про ссылки из самих статей. Согласен, можно открывать их в новом окне. Только у них нужно стиль добавить подчеркивания dashed или dotted.

**Comment by batyshkaLenin** [comment]
@Sharzon то что вы описываете это пользовательская привычка и насколько я понимаю их нужно либо реализовывать, либо предлагать другой вариант, который будет так же успешно воспринят как устоявшийся

**Comment by Sharzon** [comment]
@batyshkaLenin верно, это привычка, и ты не думаешь о том, что это будет работать как-то по-другому. Именно поэтому я написал о раздражении. Я сомневаюсь, что я единственный, кто может подобное может испытывать. Мне лично не хочется, чтобы Разрабы вызывали у кого-то раздражение, если этотне связано с самим текстом статьи. Честно, я не знаю, какую альтернативу тут можно предложить.

**Comment by philippranzhin** [comment]
Так и получается, что половина привыкла к одному поведению, другая — к другому

И тут или жделать расширенные конфиги, или раздражать пол аудитории

**Comment by batyshkaLenin** [comment]
В таком случае решают прецеденты по профильным ресурсам где сидит потенциальная аудитория.

**Comment by philippranzhin** [comment]
У хабра открывают вместо хабра, у тж vc и дтф открывает новую, у остальных тоже у кого как

Лично мне больше нравится когда новую открывает

**Comment by KOPYTOV619** [comment]
В комментариях тоже желательно сделать ссылки открывающимися в новой вкладке. Сейчас, если пользователь случайно нажал на ссылку, то происходит переход на новую страницу. Для возврата к комментарию ему необходимо сделать лишние действия, т.е. нажать назад, открыть комментарии, пролистать до того момента, где была ссылка. Если же сделать открытие всех ссылок, которые ведут на внешние ресурсы, в новых вкладках, то пользователь просто закроет вкладку и все, не надо будет выполнять лишние действия из-за случайного нажатия
